webapp/javascript/pages/tagExplorer/components/PieChart/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import ReactFlot from 'react-flot';
+import { TimelineGroupData } from '@webapp/components/TimelineChart/TimelineChartWrapper';
+import styles from './styles.module.scss';
+import { calculateMean } from '../../../math';
+
+const PieChart = ({ data }: { data: TimelineGroupData[] }) => {
```

Comment:
I think this can be a generic/reusable `PieChart` component, and the transformations are done outside it.